.github/workflows/cypress_workflow.yml

```diff
@@ -9,31 +9,29 @@ on:
   workflow_dispatch:
     inputs:
       test_repo:
-        description: 'Test repo'
+        description: 'Cypress test repo'
         default: 'opensearch-project/opensearch-dashboards-functional-test'
         required: true
         type: string
       test_branch:
-        description: 'Test branch (default: source branch)'
+        description: 'Cypress test branch (default: source branch)'
         required: false
         type: string
-      pr_number:
-        description: 'PR Number'
-        required: false
-        type: number
       specs:
-        description: 'Tests to run (default: core)'
-        default: 'cypress/integration/core-opensearch-dashboards/opensearch-dashboards/**/*.js,'
+        description: 'Tests to run (default: osd:ciGroup)'
         required: false
         type: string
+      pr_number:
+        description: 'PR Number (optional)'
+        required: false
+        type: number
 
 env:
   TEST_REPO: ${{ inputs.test_repo != '' && inputs.test_repo || 'opensearch-project/opensearch-dashboards-functional-test' }}
   TEST_BRANCH: "${{ inputs.test_branch != '' && inputs.test_branch || github.base_ref }}"
   FTR_PATH: 'ftr'
-  START_CMD: 'node ../scripts/opensearch_dashboards --dev --no-base-path --no-watch'
+  START_CMD: 'node ../scripts/opensearch_dashboards --dev --no-base-path --no-watch --savedObjects.maxImportPayloadBytes=10485760 --server.maxPayloadBytes=1759977 --logging.json=false --data.search.aggs.shardDelay.enabled=true'
```

Comment:
@manasvinibs we should consider be able to distributing a common config like selenium tests.